src/common/partialRestore.c

```diff
@@ -0,0 +1,62 @@
+#include "build.auto.h"
+#include "partialRestore.h"
+#include "postgres/interface/static.vendor.h"
+
+static int
+tableComparator(const Table *const a, const Table *const b)
+{
+    if (a->spcNode != b->spcNode)
+        return a->spcNode > b->spcNode ? 1 : -1;
+
+    if (a->relNode == b->relNode)
+        return 0;
+
+    return a->relNode > b->relNode ? 1 : -1;
+}
+
+FN_EXTERN __attribute__((unused)) List *
+buildFilterList(JsonRead *const json)
+{
+    List *result = lstNewP(sizeof(DataBase), .comparator = lstComparatorUInt);
+
+    jsonReadArrayBegin(json);
+
```

Comment:
Let's delete this empty line, because there is no empty line before jsonReadArrayEnd